module/Hydro/user.py

```
@@ -23,8 +23,17 @@ def infer_qq(html, mail) -> str:
 
 def fetch_user(config: Config, uid: str) -> UserData | None:
     logging.info("开始获取用户记录")
+    # 避免普通用户被某些插件干 403
+    if "session" not in config.get_config() or config.get_config()["session"] is None:
+        raise Exception("登录信息无效，请重试")
```

Ruff (0.14.14)

[warning] Create your own exception

(TRY002)